### PR TITLE
docs(posts): add posts feature guide

### PR DESCRIPTION
**Pull request by aditya-radhakrishnan**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/posts.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Posts
+
+<FeatureAvailability/>
+DataHub allows users to make Posts that can be displayed on the app. Currently, Posts are only supported on the Home Page, but may be extended to other surfaces on the app in the future. Posts can be used to accomplish the following:
```
Comment (chriscollins3456) [inline docs/posts.md:6]: maybe change "surface on the app" to "surface of the app"?

File: docs/posts.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+* Allowing Admins to post announcements on the home page
+* Pinning important DataHub assets or pages
+* Pinning important external links
+
+##  Posts Setup, Prerequisites, and Permissions
+
+Anyone can view Posts on the home page. To create Posts, a user must either have
+must have the **CREATE_GLOBAL_ANNOUNCEMENTS** Privilege, or possess the **Admin** DataHub Role.
```
Comment (chriscollins3456) [inline docs/posts.md:15]: remove the second "must have"

**Review by chriscollins3456 — COMMENTED** [review]
Nice! two real quick changes and this is good to go

File: docs/posts.md (hunk 1 above)
Comment (aditya-radhakrishnan) [reply docs/posts.md:6]: Makes sense

File: docs/posts.md (hunk 2 above)
Comment (aditya-radhakrishnan) [reply docs/posts.md:15]: Done

**Review by chriscollins3456 — APPROVED** [review]

**Pull request merged**